docs/astro/src/content/docs/guide/language/concepts/slint-language.mdx

```diff
@@ -4,4 +4,150 @@ title: Slint Language
 description: Slint Language
 ---
 
-Component basics. It's components all the way down...
\ No newline at end of file
+import Link from '/src/components/Link.astro';
+
+The concepts section of the guide are to give you an overview of the language from a high level. If you want to
+dive straight in then the details then the <Link type="slintFile" label="coding section"/> and language reference
+sections are the best places to start.
+
+This section is to give you an insight into the thinking behind the language and the core concepts it is made up from.
+
+
+
+## Why Slint?
+
+The Slint language helps you to describe the User Interface of your application in a declarative way.
+
+The reasons for this are that first a User Interface is quite different from abstract code. It's made up 
```

Comment:
I'd go straight for the bold statement. Otherwise I ask myself: "Reasons for what?" and "What's the second, after first"? :-)


```suggestion
A User Interface is quite different from abstract code. It's made up 
```